### ISSUE TITLE
Problem displaying images added to a product as attribute

### ISSUE THREAD

**Issue by marcosbc** [akeneo/pim-community-dev#4062]
Hi all,

I've found an issue that happens when you add a catalog image attribute to a product and try to display it. It is also **reproducible in the Akeneo demo** at: https://www.akeneo.com/demo/
I'm using Akeneo 1.4.20.

Steps to reproduce:
1. Create a sample catalog image attribute.
2. Create a new product.
3. Add the catalog image attribute to the product, upload an image. You will see the following:
![screen shot 2016-02-26 at 12 57 56](https://cloud.githubusercontent.com/assets/5102866/13352071/28cb1d40-dc8b-11e5-8e38-422939c0cf91.png)

The image is uploaded but it is not being shown by Akeneo. Copying the link location you will see it has encoded the path to the image (`/tmp/pim/...`), but appending `/thumbnail_small` so it redirects to the default thumbnail. If you remove the `/thumbnail_small` text and try to access it, you will get a HTTP 500 Internal Server Error code.

This is the entry that is added for each request in my `prod.log`:
```
[2016-02-26 03:06:42] request.INFO: Matched route "pim_enrich_media_show". {"route_parameters":{"_controller":"pim_enrich.controller.file:showAction","filter":null,"filename":"%2Ftmp%2Fpim%2Ffile_storage%2F7%2F0%2Fd%2Fc%2F70dc3d60c06fc6a5c1b62e645dfe8af719c2ea3c_Screen_Shot_2016_02_26_at_11.45.56.png","_route":"pim_enrich_media_show"},"request_uri":"http://127.0.0.1/media/show/%252Ftmp%252Fpim%252Ffile_storage%252F7%252F0%252Fd%252Fc%252F70dc3d60c06fc6a5c1b62e645dfe8af719c2ea3c_Screen_Shot_2016_02_26_at_11.45.56.png"} []
```

System info text report: [akeneo-pim-system-info_2016-02-26_04-07.txt](https://github.com/akeneo/pim-community-dev/files/148151/akeneo-pim-system-info_2016-02-26_04-07.txt)

**Comment by nidup** [comment]
Hi @marcosbc 

Thank you for the feedback!

In fact the thumbnail is only computed and displayed when you save the product, the feature has been built this way but I do agree that this could be enhanced to display it just after the upload.

**Comment by nidup** [comment]
Hi @marcosbc !

This improvement has been released in v1.5.4 (duplicated in this issue https://github.com/akeneo/pim-community-dev/issues/3553).

Have a good day!

**Comment by oriolmatamoros** [comment]
Hello, I'm having the same issue with akeneo 4.0 CE. When I upload an image. The image is uploaded but it is not being shown by Akeneo. If I try to access it, I will get a HTTP 500 Internal Server Error code. 

Can anybody help?

Thanks!
